src/components/pages/AdvicesPage/components/AdvicesList/AdviceItem.tsx

```
@@ -57,24 +56,24 @@ const AdviceItem: FC<AdviceItemProps> = ({
       <MobileHidden>
-        <AdvTitle>{item.attributes.title}</AdvTitle>
+        <AdvTitle>{item.title}</AdvTitle>
       </MobileHidden>
       <MobileVisible>
-        <AdvTitle>{item.attributes.title}</AdvTitle>
+        <AdvTitle>{item.title}</AdvTitle>
       </MobileVisible>
-      {!!adviceClicked ? (
+      {adviceClicked ? (
         <AdvDescription
           dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)